velox/docs/functions/presto/datetime.rst

```diff
@@ -188,6 +188,10 @@ Date and Time Functions
 
         SELECT current_timezone;   -- Asia/Kolkata
 
+.. function:: current_time() -> time with time zone
```

Comment:
This function is alphabetically before current_timezone. Please move it ahead of it.

Reply:
Done.